tests/automated_install.sh

```diff
@@ -59,11 +66,18 @@ printf "#### Setup VB's and Bootstrap\n"
 source ./vbox_create.sh
 
 # Ensure we got VM_LIST from vbox_create.sh
-echo "Using CLUSTER_PREFIX=${BACH_CLUSTER_PREFIX} VM_LIST=${VM_LIST}"
+echo "Using CLUSTER_PREFIX=${BACH_CLUSTER_PREFIX}"
+echo "VM LIST"
```

Comment:
`$VM_LIST` echo number one; is this useful? (If so some context would help users.)

Reply:
Which is why it is followed by
https://github.com/pu239ppy/chef-bach/blob/3765002dc29acb3305f3d8740e11fcebdba17239/tests/automated_install.sh#L71-L73